client/my-sites/domains/domain-management/components/primary-domain-selector/style.scss

```diff
@@ -0,0 +1,17 @@
+.domains-set-primary-address {
+	&.form-fieldset {
+		margin-bottom: 2em;
+		box-shadow: 0 0 0 1px var(--color-border-subtle);
+		padding: 24px;
+	}
+	&__title.form-label {
+		margin-bottom: 0.5em;
+		font-size: 1rem;
+		font-weight: 600;
+	}
+	&__subtitle.form-setting-explanation {
+		font-style: normal;
+		margin-bottom: 1em;
+		color: var(--color-text-subtle);
```

Comment:
Another nit: colour here isn't needed since it's inherited part of the component. 

```suggestion
```